MAUI/ImageEditor/customview.md

```diff
@@ -0,0 +1,208 @@
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Clicked="OnAddCustomAnnotationViewClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnAddCustomAnnotationViewClicked(object sender, EventArgs e)
+    {
+        Image customImage = new Image() { HeightRequest = 100, WidthRequest = 100, Aspect= Aspect.Fill};
+        customImage.Source = ImageSource.FromFile("emoji.png");
+        this.imageEditor.AddCustomAnnotationView(customImage, 
+            new Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings 
+            { 
+                Bounds = new Rect(0.2, 0.2, 0.5, 0.1), 
+                RotationAngle = 45,
+                AllowResize = true,
+                AllowDrag = false
+            });
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Delete the custom view
+
+Delete the selected custom view using the [`DeleteAnnotation`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_DeleteAnnotation) method.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Text="DeleteAnnotation"
+                Clicked="OnDeleteAnnotationClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnDeleteAnnotationClicked(object sender, EventArgs e)
+    {
+        this.imageEditor.DeleteAnnotation();
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Clear all custom view
+
+Remove all the custom view annotations using the [`ClearAnnotations`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ClearAnnotations) method.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Text="ClearAnnotations"
+                Clicked="OnClearAnnotationsClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnClearAnnotationsClicked(object sender, EventArgs e)
+    {
+        this.imageEditor.ClearAnnotations();
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Annotation selected event
+
```

Comment:
We have included annotation unselected event, mention that in all annotations UG

Reply:
annotation unselected content updated in all annotations UG.